static/js/publisher/market/market.js

```diff
@@ -111,9 +111,8 @@ function initForm(config, initialState, errors) {
     }
   });
 
-  document.querySelector('.js-market-submit').addEventListener('click', function(event) {
+  marketForm.addEventListener('submit', function(event) {
```

Comment:
Needed to move this to native submit to keep browser validation. Submitting from JS was causing HTML validation to be skipped.